subprojects/packagefiles/glfw/meson_options.txt

```diff
@@ -0,0 +1,26 @@
+#####  GLFW build options
+
+#@ Build options
+option('build-docs',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW documentation?')
+option('build-tests',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW test programs?')
+option('build-examples',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW example programs?')
```

Comment:
Looking at other wraps it seems that the most common spelling for this kind of options is `tests` and `examples` without the `build-` prefix, in line with the Meson style recommendation to not name options with the `enable-` prefix. It may be nice to standardize on the same name for options with the same intent.

Reply:
This is true. I'll get those changed.